cmd/oras/root/pull.go

```diff
@@ -206,7 +206,7 @@ func doPull(ctx context.Context, src oras.ReadOnlyTarget, dst oras.GraphTarget,
 			}
 			defer func() {
 				if fetchErr != nil {
-					rc.Close()
+					_ = rc.Close()
```

Comment:
Looks like this can be tested by mocking src.Fetch() to return an error producing rc?

Reply:
True.